src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -977,10 +977,12 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 			return;
 		}
 
+		const title = localize('chat.attachment.clearButton', "Remove from context");
 		const clearButton = new Button(widget, {
 			supportIcons: true,
 			hoverDelegate,
-			title: localize('chat.attachment.clearButton', "Remove from context"),
+			title,
+			ariaLabel: title
```

Comment:
You can also set `ariaLabel: true` so you don't have to duplicate it

Reply:
Why wouldn't that be true by default?

Reply:
that doesn't work in this case because `label` isn't set , which is where that is applied 

https://github.com/microsoft/vscode/blob/e3d033cf31ffba88c2185b1f79f6e57caea39fa3/src/vs/base/browser/ui/button/button.ts#L264-L269